src/libraries/System.Private.CoreLib/src/System/Text/Latin1Encoding.Sealed.cs

```diff
@@ -0,0 +1,29 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Text
+{
+    internal partial class Latin1Encoding : Encoding
+    {
+        /// <summary>
+        /// A special instance of <see cref="Latin1Encoding"/> that is initialized with "don't throw on invalid sequences;
+        /// use best-fit substitution instead" semantics. This type allows for devirtualization of calls made directly
+        /// off of <see cref="Encoding.Latin1"/>.
+        /// </summary>
+        internal sealed class Latin1EncodingSealed : Latin1Encoding
```

Comment:
why do we call it sealed? Can't we call it BestFitLatine1Encoding (including the file name of course)?

Reply:
Uses same naming pattern as `UTF8Encoding` and `ASCIIEncoding`.